snuba/query/timeseries_extension.py

```diff
@@ -1,19 +1,59 @@
 from datetime import datetime, timedelta
 from typing import Any, Mapping, Tuple
 
-from snuba import state
-from snuba.util import parse_datetime
+from snuba import environment, state
 from snuba.query.conditions import (
-    binary_condition,
     BooleanFunctions,
     ConditionFunctions,
+    binary_condition,
```

Comment:
do we usually make distinction between functions and classes when sorting imports? or are we case sensitive on sorting ?

Reply:
I've just been using the vscode "Sort Imports" macro which is case sensitive (at least by default.) I personally don't have a strong opinion either way, happy to do whatever makes sense and is easiest as long as it's consistent.

Reply:
I am ok with either as long as we all know the conventions we use.